src/apprt/gtk/close_dialog.zig

```diff
@@ -0,0 +1,149 @@
+const std = @import("std");
+
+const App = @import("App.zig");
+const Window = @import("Window.zig");
+const Tab = @import("Tab.zig");
+const Surface = @import("Surface.zig");
+const adwaita = @import("adwaita.zig");
+
+const gobject = @import("gobject");
+const gio = @import("gio");
+const adw = @import("adw");
+const gtk = @import("gtk");
+
+pub const CloseDialog = extern struct {
```

Comment:
I want to see more comments in this file, at the very least this structure and what its purpose is but the private data and so on could also use docs.

Reply:
No I understand that, I mean the higher level docs on what this is and what fields should be set and so on. Just general structure docs.

Reply:
I should add, just more generally, the coding style I have is _very comment heavy_. There are reasons for this and I can explain them another time, but making sure the code stays idiomatically the same throughout the project is important. 😄